Sources/Core/Structure/Web3ProviderProtocol.swift

```diff
@@ -5,6 +5,9 @@
 //  Created by Yaroslav Yashin on 11.07.2022.
 //
 
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
```

Comment:
In [this proposal](https://forums.swift.org/t/pitch-move-urlsession-to-new-foundationnetworking-module/14002) I see that URL doesn't included within `FoundationNetwork`, is it necessary here?

Reply:
Let me double check if this one is necessary